src/components/RootNavigation.js

```diff
@@ -3,35 +3,15 @@ import PropTypes from 'prop-types';
 import { NavItem } from '@newrelic/gatsby-theme-newrelic';
 
 const RootNavigation = ({ nav }) => {
-  const tdp = nav.pages.find(
-    (page) => page.title === 'Telemetry Data Platform'
-  );
-  const fso = nav.pages.find(
-    (page) => page.title === 'Full-Stack Observability'
-  );
-  const ai = nav.pages.find(
-    (page) => page.title === 'Alerts and Applied Intelligence'
-  );
-
-  const pages = nav.pages.filter((page) => ![tdp, fso, ai].includes(page));
-
   return (
     <nav role="navigation" aria-label="Navigation">
-      <NavItem page={tdp} />
-      <NavItem page={fso} />
-      <NavItem page={ai} />
       <hr />
```

Comment:
I think we want to get rid of this `<hr />`, it's adding an extra section break to the side bar.
```suggestion
```
<img width="283" alt="Screen Shot 2021-10-14 at 1 37 22 PM" src="https://user-images.githubusercontent.com/70179303/137393070-3febbdee-290c-4f60-8ce8-0cbfd26c8dce.png">